Makefile

```diff
@@ -81,27 +82,42 @@ vet:
 # Run go lint.
 lint:
 	golint || go get -u golang.org/x/lint/golint
-	golint -set_exit_status ./cmd/... ./pkg/... ./clusterctl/...
+	golint -set_exit_status ./cmd/... ./pkg/...
 
 # Build the docker image
 docker-build: generate fmt vet
-	docker build . -t ${IMG}
+	docker build . -t ${MANAGER_IMAGE}
 
 # Push the docker image
 docker-push:
-	docker push ${IMG}
+	docker push ${MANAGER_IMAGE}
 
 # Cleanup
 clean:
-	rm -rf out
+	rm -rf cmd/clusterctl/examples/aws/out/
 	rm -f kubeconfig
 	rm -f minikube.kubeconfig
 
-manifests:
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go rbac --name aws-manager
+# Manifests.
+cmd/clusterctl/examples/aws/out/:
+	./cmd/clusterctl/examples/aws/generate-yaml.sh
+
+cmd/clusterctl/examples/aws/out/credentials: cmd/clusterctl/examples/aws/out/ clusterawsadm
+	clusterawsadm alpha bootstrap generate-aws-default-profile > cmd/clusterctl/examples/aws/out/credentials
```

Comment:
Don't think `cmd/clusterctl/examples/aws/out` is a good place to place the credentials as it is not part of an example.
Suggest creating like a `build.out` directory to store the manifests. WDYT?

Reply:
The clusterctl command always references the out directory in the example. We should probably revisit this later on when the example grows.

Reply:
We should change that. IMO, the credentials file doesn't really belong in the `examples` dir. We don't want people leaking their creds  by mistake.

Reply:
Could we track / fix that as a follow-up item or is this a blocking issue?

Reply:
Lets track as a followup item.